test/e2e/app-dir/app-invalid-revalidate/app-invalid-revalidate.test.ts

```diff
@@ -0,0 +1,78 @@
+import { createNextDescribe } from 'e2e-utils'
+import { check } from 'next-test-utils'
+
+createNextDescribe(
+  'app-invalid-revalidate',
+  {
+    files: __dirname,
+    skipStart: true,
```

Comment:
how come we skip `start`?

Reply:
we don't need the initial build as we trigger it after modifying the specific assertions.